extensions/amp-story/1.0/amp-story.js

```diff
@@ -1820,6 +1820,8 @@ export class AmpStory extends AMP.BaseElement {
           'amp-story-page amp-story-page-attachment'
         );
 
+        this.initializeBookend_().then(() => this.showBookend_());
```

Comment:
Could having this outside of the `this.vsync_.mutate` below cause race conditions that would result in rendering the bookend sometimes before the page attachments, and sometimes after?

Reply:
The `initializeBookend_` method creates the bookend synchronously, if it didn't exist yet. It'll always be before the page attachments